campaigns/mobile/mobile-snippet-widget.html

```diff
@@ -0,0 +1,420 @@
+      display: block;
+      visibility: hidden;
+      height: 0;
+      clear: both;
+      content: ".";
+    }
+
+    #send-to-device-form-{{ snippet_id }} .input .legal {
+      font-size: 10px;
+      max-width: 38em;
+      margin: 10px auto;
+      clear: both;
+    }
+
+    #send-to-device-form-{{ snippet_id }} .thank-you {
+      position: relative;
+      padding-top: 92px;
+      font-size: 16px;
+    }
+
+    #send-to-device-form-{{ snippet_id }} .thank-you:before {
+        position: absolute;
+        top: 0;
+        left: 50%;
+        content: '';
+        width: 72px;
+        height: 72px;
+        margin-left: -36px;
+        background-image: url("https://www.mozilla.org/media/img/send-to-device/device-icon.28cef022bba2.png");
+    }
+
+    #send-to-device-form-{{ snippet_id }} .thank-you a {
+        font-size: 12px;
+    }
+
+    #send-to-device-form-{{ snippet_id }} a.more:after,
+    #send-to-device-{{ snippet_id }} footer ul .store a:after {
+      content: "\00A0\00BB";
+      white-space: nowrap;
+    }
+
+    #send-to-device-{{ snippet_id }}.logo footer {
+      border-bottom-left-radius: 10px;
+      border-bottom-right-radius: 10px;
+    }
+
+    #send-to-device-{{ snippet_id }} footer {
+      margin-top: 20px;
+      padding: 10px 0;
+      background: #e1e2e2;
+      border-top: 1px solid #d3d4d4;
+      zoom: 1;
+    }
+
+    #send-to-device-{{ snippet_id }} footer ul .store{
+      float: none;
+      width: 100%;
+      text-align: center;
+      color: #B0B0B0;
+    }
+
+    #send-to-device-{{ snippet_id }} footer ul li a {
+      font-size: 14px;
+      line-height: 2;
+    }
+
+  </style>
+
+  <script type="text/javascript">
+   //<![CDATA[
+   (function() {
+       var snippet = document.getElementById('firefox-mobile-activation-{{ snippet_id }}');
+       snippet.addEventListener('show_snippet', function() {
+           var input = document.querySelector('#firefox-mobile-activation-{{ snippet_id }} .number');
+           var button = document.querySelector('#firefox-mobile-activation-{{ snippet_id }} .submit-button');
+           var input_type;
+
+           // Checking input
+           input.addEventListener('input', function() {
+               input_type = checkInput(input.value);
+               if (input_type) {
+                   button.disabled = false;
+                   button.classList.remove('insensitive');
+               }
+               else {
+                   button.disabled = true;
+                   button.classList.add('insensitive');
+               }
+               console.log('checking ' + input.value + ' = ' + input_type);
+           });
+
+          if ("createEvent" in document) {
+              var evt = document.createEvent("HTMLEvents");
+              evt.initEvent("input", false, true);
+              input.dispatchEvent(evt);
+          }
+          else
+              input.fireEvent("input");
```

Comment:
We do not need to check support here, we can just go ahead and use `createEvent`